pkgs/applications/graphics/plasma-camera/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, mkDerivation
+, fetchFromGitLab
+
+, cmake
+, extra-cmake-modules
+, wrapGAppsHook
+
+, gst-plugins-bad
+, gst-plugins-base
+, gst-plugins-good
+, gstreamer
+, kconfig
+, kcoreaddons
+, ki18n
+, kirigami2
+, qtmultimedia
+, qtquickcontrols2
+}:
+
+mkDerivation rec {
+  pname = "plasma-camera";
+  # The last tagged version is more than a year old and slightly broken
+  version = "2021-03-02";
+
+  src = fetchFromGitLab {
+    domain = "invent.kde.org";
+    owner = "plasma-mobile";
+    repo = pname;
+    rev = "d09ec725a6e89128d72c2c58d1a433744a867733";
+    sha256 = "082j4xvbx1y7glziwsc3zlvgixhs7pnfbac9lqjdn0kjibycy12b";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    extra-cmake-modules
+    wrapGAppsHook
```

Comment:
This will double wrap. Better add to `qtWrapperArgs`.

Reply:
I'm not convinced that it's better. In fact I'm not convinced peppering wrappers everywhere is a solution to any kind of problem we're having here :/.

I'm not sure what to add, but I assume we'd copy some of the logic of `wrapGAppsHook` in `qtWrapperArgs`. Wouldn't that risk drifting from the main implementation and maybe break?

Any documentation for combining `GAppsHook` and `qtWrapper`?

Reply:
`qtWrapperArgs+=("''${gappsWrapperArgs[@]}")` should do the trick.